src/main/webapp/src/app/create-gardens-form/create-gardens.component.css

```diff
@@ -0,0 +1,20 @@
+.button {
```

Comment:
What do you think about having some of the pages share a CSS file or adding this rule to a global file? Maybe that will help dedup some of these rules.

Reply:
FYI, I already have a common style page in `.../src/app/common/growpod-page-styles.css` -- you may need to modify your `create-gardens.component.ts` to look for this file.